### PR TITLE
Add date pipe docs

### PR DESCRIPTION
**Pull request by blackbaud-johnly**

### CODE REVIEW

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux?_sp.spa=pr-add-date-pipe-docs](https://developer.blackbaud.com/skyux?_sp.spa=pr-add-date-pipe-docs)

File: src/app/components/date-pipe/index.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<sky-docs-url-redirect
```
Comment (Blackbaud-BobbyEarl) [inline src/app/components/date-pipe/index.html]: Do we need a redirect here if it's never existed?

File: src/app/components/demo-components.service.ts [hunk 2]
```diff
@@ -195,6 +195,12 @@ export class SkyDemoComponentsService {
         getCodeFiles: () => this.getDemoFiles('Copy to clipboard'),
         disableStackblitz: true
       },
+      {
+        name: 'Date pipe',
+        icon: 'calendar',
+        summary: `The date pipe formats date values according to locale rules.`,
+        url: '/components/date-pipe'
```
Comment (Blackbaud-BobbyEarl) [inline src/app/components/demo-components.service.ts]: Same question as above, I believe it's safe to ignore adding here as this service is no longer used.  I can confirm that though, and if it's true, probably remove this service all together.

Comment (blackbaud-johnly) [reply src/app/components/demo-components.service.ts]: Good point. Thanks for the offline explanation; I was confused and thought we needed the `date-pipe` folder in `skyux2-docs` to make this work. I'm closing this PR and making the updates we discussed in the `sky-skysp-svc` repo.

**Pull request closed without merging**